samples/sensor/bme280/app.overlay

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2020, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * By default, this overlay sets up a single BME280 sensor
+ * on the standard Arduino header I2C pins. Adjust the aliases
+ * definition below to switch to SPI.
+ *
+ * If your board's devicetree has "arduino_i2c" and "arduino_spi" node
+ * labels, you can use this overlay as-is, wiring the BME280 sensor to
+ * the breakout header of your choice accordingly:
+ *
+ * https://upload.wikimedia.org/wikipedia/commons/c/c9/Pinout_of_ARDUINO_Board_and_ATMega328PU.svg
+ *
+ * If you're not sure about your board, just compile the application and see.
+ * You'll get a build error if there is no "arduino_i2c" or "arduino_spi".
+ *
+ * If your board does not have an Arduino header, you can create a file
```

Comment:
I'm not sure about this.  `ruuvi_ruuvitag` in-tree has a BME280 in its devicetree, and does not have Arduino header support.  Builds of this sample for it fail because the app.overlay gets picked up, then fails to parse because the arduino bus labels aren't defined.

I get past that with `touch boards/ruuvi_ruuvitag.overlay` to trivially bypass the inappropriate generic overlay, but then the sample doesn't work because there's no `sample_bme280` alias.  Comment on that elsewhere.